pkg/util/gkenetworkparamsetutil/BUILD

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "gkenetworkparamsetutil",
+    srcs = [
+        "controller_utils.go",
+        "update_utils.go",
+    ],
+    importpath = "k8s.io/cloud-provider-gcp/pkg/util/gkenetworkparamsetutil",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//vendor/k8s.io/apimachinery/pkg/apis/meta/v1:meta",
+        "//vendor/k8s.io/apimachinery/pkg/util/runtime",
+        "//vendor/k8s.io/cloud-provider-gcp/crd/apis/network/v1alpha1",
```

Comment:
Please don't workaround https://github.com/kubernetes/cloud-provider-gcp/issues/384 with this hack.

Reply:
In this commit, crd was added to deps under the vendor directory already. https://github.com/kubernetes/cloud-provider-gcp/commit/3690a647cabf0173ced933b7c49a6cd7f296c190

Should we let this go through then follow up fixing the crd import path in another pr?

Reply:
#430 fixes #384.

Added #431 to understand how we should move forward with `/crd`.

LGTM for using vendor in current form for now. However, please be aware that it's pinned to specific version and does not receive any upgrades.